packages/backend/src/db/storage/schedule-reminders.ts

```diff
@@ -8,6 +8,7 @@ export const SCHEDULE_REMINDERS_TEMPLATE: Template = {
   description:
     'Schedule a recurring reminder to yourself to complete a task everyday',
   iconName: 'BiCalendar',
+  tag: 'empty',
```

Comment:
tags

Reply:
same thing gonna do it in the pr where i combine demos and templates